addons/survey/views/survey_report_templates.xml

```diff
@@ -1,122 +0,0 @@
-                             </p>
-                        </div>
-                        <div t-else="" class="certification-failed">
-                            <p>Certification Failed</p>
-                        </div>
-                    </div>
-
-                    <div class="certification-bottom">
-                        <div class="certification-date-wrapper">
-                            <div class="certification-date" t-field="user_input.create_date" t-options='{"widget": "date"}'/>
-                            <span>Date</span>
-                        </div>
-                        <div class="certification-company">
-                            <span class="certification-company-logo" t-field="user_input.env.company.logo" t-options="{'widget': 'image'}" role="img"/>
-                        </div>
-
-                    </div>
-                    <div class="certification-number" t-if="user_input.scoring_success">
-                        Certification n°<t t-esc="str(user_input.id).rjust(10, '0')"/>
-                    </div>
-                </div>
-                <div t-if="user_input.test_entry" class="test-entry"/>
-            </div>
-        </template>
-
-        <template id="certification_report_view_classic">
-            <!-- Style classes to be applyed to '#o_survey_certification': [no class](purple), gold, blue  -->
-            <div id="o_survey_certification" t-att-data-oe-model="user_input._name" t-att-data-oe-id="user_input.id" t-att-class="'article certification-wrapper classic %s' % layout_color">
-                <div t-if="user_input.test_entry" class="test-entry"/>
-                <div class="certification">
-                    <div class="certification-top">
-                        <h1><b>Certificate</b>
-                            <br/><span t-if="user_input.scoring_success">of achievement</span>
-                        </h1>
-                    </div>
-
-                    <div class="certification-content">
-                        <div t-if="user_input.scoring_success">
-                            <p>This certificate is presented to
-                                 <t t-set="certif_style" t-value="''"/>
-                                <t t-set="certified_name" t-value="user_input.partner_id.name or user_input.email or ''"/>
-                                <t t-if="certified_name.isupper()">
-                                    <t t-set="certif_style" t-value="certif_style + 'font-family: certification-serif;'"/>
-                                </t>
-                                <t t-if="len(certified_name) > 35">
-                                    <t t-set="certif_style" t-value="certif_style + 'font-size: 20px; line-height: 4; font-family: certification-serif; '"/>
-                                </t>
-                                <t t-elif="len(certified_name) > 20">
```

Comment:
I've been playing the "7 differences game" but it's hard to see what are the differences between the 2 layouts, that would justify that we had 2 before (and only 1 in your PR).

Did you notice any?

Reply:
There were 3-4 small changes(placement of certification number, change in style for length of certified name and the badge), which I changed by either adding "layout_template" conditions or by making small changes into css. For now I believe that this template looks identical to the ones that are currently on runbot